lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -88,6 +90,10 @@ def configure(conf)
       @remove_keys_on_update = @remove_keys_on_update.split ','
     end
 
+    if @logstash_prefix_key && @logstash_prefix_key.is_a?(String)
+      @logstash_prefix_key = @logstash_prefix_key.split '.'
```

Comment:
I'm planning to support this feature with v0.14's [extract_placeholders](https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/output.rb#L686).
Could you use this method to extract placeholders?
This is more generic way in v0.14/

Reply:
#288